bumpversion/__init__.py

```diff
@@ -317,13 +318,18 @@ class MercurialDoesNotSupportSignedTagsException(Exception):
     def __init__(self, message):
         self.message = message
 
+class UnkownPart(Exception):
+    def __init__(self, message):
+        self.message = message
+
 def keyvaluestring(d):
     return ", ".join("{}={}".format(k, v) for k, v in sorted(d.items()))
 
 class Version(object):
 
-    def __init__(self, values, original=None):
+    def __init__(self, values, config, original=None):
```

Comment:
I'm not a fan of the config here. It doesn't feel very logical that a version has a config, especially if it's only used for order.

Reply:
Agreed, will only use the order